lib/tdf3/src/client/BrowserTdfSteam.ts

```diff
@@ -22,6 +22,8 @@ export class BrowserTdfStream extends DecoratedReadableStream {
       return;
     }
 
+    streamSaver.mitm = 'https://secure.virtru.com/rca/stream-saver/index.html';
```

Comment:
Any place we can un-hardcode this and make it dynamic per environment? Although Auth-Widget is hardcoded to prod so maybe this isn't a deal breaker?

Reply:
no Virtru urls in OpenTDF

Reply:
Any opinions on where this should live? Since stream-saver is only utilized in client-web?